acceptance-tests/apps/mssqlapp/internal/app/app.go

```diff
@@ -17,20 +17,20 @@ const (
 	valueColumn = "valuedata"
 )
 
-func App(config string) *mux.Router {
+func App(config string) http.Handler {
 	db := connect(config)
 	defer db.Close()
 	if err := db.Ping(); err != nil {
 		log.Fatalf("failed to ping database: %s", err)
 	}
 
-	r := mux.NewRouter()
-	r.HandleFunc("/", aliveness).Methods("HEAD", "GET")
-	r.HandleFunc("/{schema}", handleCreateSchema(config)).Methods("PUT")
-	r.HandleFunc("/{schema}", handleFillDatabase(config)).Methods("POST")
-	r.HandleFunc("/{schema}", handleDropSchema(config)).Methods("DELETE")
-	r.HandleFunc("/{schema}/{key}", handleSet(config)).Methods("PUT")
-	r.HandleFunc("/{schema}/{key}", handleGet(config)).Methods("GET")
+	r := chi.NewRouter()
+	r.Head("/", aliveness)
```

Comment:
Missing Get for "/". Unsure if needed. Feel free to ignore.

Reply:
As you say, it's never used, and didn't seem worth adding an extra line for.